### PR TITLE
UI: Remove custom service

### PR DESCRIPTION
**Pull request by hashishaw**
Fix a test that was failing because we have a service that overrides a service of the same name.

### CODE REVIEW

File: ui/app/services/flash-messages.ts
Comment (hellobontempo) [inline ui/app/services/flash-messages.ts:1]: because the external package isn't typescript, we need to keep the declaration file, right?

**Review by hellobontempo — APPROVED** [review]
Thanks for tackling!

File: ui/app/services/flash-messages.ts
Comment (hashishaw) [reply ui/app/services/flash-messages.ts:1]: This was the definition file not the declaration file, so it does get removed

Comment (hellobontempo) [reply ui/app/services/flash-messages.ts:1]: That was my question! Since I didn't see it among the deleted files 🤔

**Comment by hellobontempo** [comment]
For more context, when we added the typescript declaration file to the pki ember engine, we changed the name from `FlashMessage` to `FlashMessageService`  [here](https://github.com/hashicorp/vault/pull/18146/files#diff-5ebd44ba5909c95c331fd105e463a0332fa06bfc2f1244a4324d2518c29d9e59R3). I'm not sure why it took so long for the failure to catch up (5 months 🤯 ), but _something changed_ and the flash messages in the main app starting calling the default service, instead of our custom one, resulting in the following error:
```
Transform secrets: it shows a message if an update fails after save
global failure: TypeError: this.flashMessages.stickyInfo is not a function
```

image

**Pull request merged**